about.html

```diff
@@ -0,0 +1,135 @@
+          <li class="nav_menu_item close_button">&times;</li>
+          <li class="nav_menu_item">
+            <a class="other_item" href="./index.html">Home</a>
+          </li>
+          <li class="nav_menu_item">
+            <a class="other_item link" href="./about.html">About</a>
+          </li>
+          <li class="nav_menu_item">
+            <a class="other_item" href="#">Program</a>
+          </li>
+          <li class="nav_menu_item"><a class="other_item" href="#">Join</a></li>
+          <li class="nav_menu_item">
+            <a class="other_item" href="#">Sponsor</a>
+          </li>
+          <li class="nav_menu_item"><a class="other_item" href="#">News</a></li>
+          <li class="nav_menu_item">
+            <a class="other_item er_link" href="#">ER Exhibition</a>
+          </li>
+        </ul>
+        <div class="hamburger">
+          <img src="./images/hamburger-menu.png" alt="hambuger" />
+        </div>
+      </div>
+    </nav>
+    <section class="about">
+      <h3 class="about_sub_title">"Homes For the Masses"</h3>
+      <h1 class="about_title">AFFORDTABLE HOUSING EXHIBITION 2022</h1>
+      <p class="about_intro">
+        Exquisite Residence ehibition always provide a platforn for networking
+        between prospective housing owners, investors, financial experts,
+        marketer construction expert e.t.c.. It create synergy between between
+        all participant. This Year, we have gone a step further beyond the
+        shores of Nigeria a bring experts from different cultural background
+        with years of experience. We promise this exhibition will be worth time
+        spent.
+      </p>
+      <p class="contact">Please Contact Us Via Email for more inquiries</p>
+      <p class="email">www.affordhouses@exquisteestate.org</p>
+    </section>
+    <section class="about_exquisite">
+      <h3 class="exquisite">
+        Exquisite Residence Affordable Housing Exhibition 2022
+      </h3>
+      <div class="bar about_bar"></div>
+      <p class="logo_origin">
+        The Logo of this exhibition was created by the winner of logo
+        competition carried in the year 2021
+      </p>
+      <div class="about_logo"><img src="./images/logo.png" alt="Logo" /></div>
+    </section>
+    <section class="past_exhibition">
+      <h3 class="past_title">See Past Exquisite Residence Exhibition</h3>
+      <div class="bar about_bar"></div>
+      <p class="last_two">
+        Take a look at the last exhibition that took place in Ghana and Benin
+        Repulic
+      </p>
+      <ul class="past_exhibition_menu">
+        <li class="past_exhibition_menu_item">
+          <p class="year">2020</p>
+          <p class="year_exh">Exquisite Residence Exhibition 2020</p>
+        </li>
+        <li class="past_exhibition_menu_item">
+          <p class="year">2018</p>
+          <p class="year_exh">Exquisite Residence Exhibition 2018</p>
+        </li>
+      </ul>
+    </section>

```

Comment:
- [ ] Please apply color effect on the images and make sure it matches the Design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/169279124-584e2859-98be-4051-8b07-af8b2c778369.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/169279162-cd33dd0b-a613-49cb-8fac-c4988bdee3c1.png)